contrib/helm/hello-world/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for helloWorld.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: tutum/hello-world
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart version.
+  tag: latest
```

Comment:
Should this be pinned?

Reply:
created a separate version of hello-world with pinned tag.